src/javascripts/components/copy.js

```diff
@@ -7,18 +7,18 @@
   // This module is dependent on /vendor/clipboard.js
   GOVUK.copy = {
     init: function (selector) {
-      $(selector).parent().prepend('<a class="govuk-link app-link--copy" href="#copy" aria-live="assertive">Copy</a>')
+      $(selector).parent().prepend('<button class="copy-button" aria-live="assertive">Copy</button>')
       // Copy to clipboard
       try {
-        new ClipboardJS('.app-link--copy', {
+        new ClipboardJS('.copy-button', {
           target: function (trigger) {
             return trigger.nextElementSibling
           }
         }).on('success', function (e) {
-          e.trigger.text = 'Copied'
+          e.trigger.innerText = 'Copied'
           e.clearSelection()
           setTimeout(function () {
-            e.trigger.text = 'Copy'
+            e.trigger.innerText = 'Copy'
```

Comment:
Don't think it'll matter but `textContent` may be more appropriate here.

https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Differences_from_innerText

Reply:
TIL – thanks!